semgrep_output_v1.atd

```diff
@@ -978,13 +978,31 @@ type profile = {
       <python repr="dict">
       <ts repr="map">;
 
+    ?parsing_time : parsing_time option;
+
     targets: target_times list;
     total_bytes: int;
 
     (* maximum amount of memory used by Semgrep(-core) during its execution *)
     ?max_memory_bytes : int option;
   }
 
+type file_time <ocaml attr="deriving show"> = {
+    fpath: fpath;
+    ftime: float;
+}
+
+type stats = {
+    mean: float;
```

Comment:
We could also have a `total: float` field here. I'd be fine with that, it's just the field name `per_file_time` makes it more clear what the data is, and with that name it felt a bit stranger to include the total.

Reply:
Maybe we could rename the field to "summary_stats"?